frontend/src/dashboard/dashboard-route.tsx

```diff
@@ -9,6 +9,15 @@ const Header = styled.div`
   margin-bottom: 20px;
 `;
 
+const TopicArea = styled.div`
```

Comment:
I don't see any use for this styled component it styles are empty

Reply:
Changes have been made. Now the use of components makes more sense.